--- a/builder/symlink/symlink.go
+++ b/builder/symlink/symlink.go
@@ -90,6 +90,10 @@ func populateVendorPath(vendorPath string, src string) {
 	for _, f := range files {
 		innerSrc := filepath.Join(src, f.Name())
 		dst := filepath.Join(vendorPath, f.Name())
+		if _, err := os.Lstat(dst); err == nil {
+			continue
+		}
 		if err := os.Symlink(innerSrc, dst); err != nil {
 			fmt.Println("symlink error, trying", innerSrc, err)
 			populateVendorPath(dst, innerSrc);
